### DISCUSSION TITLE
Vision Pro Support

### DISCUSSION THREAD

**Discussion by trafalco** [blinksh/blink#1942] [Ideas]
Any plans to support Vision Pro?  It would be great to have multiple blink terminals up simultaneously.

**Comment by carloscabanero** [comment]
We do have plans to support but currently waiting for the dust to settle. We already have users on Vision Pro all the time, and they are currently helping with bugs on our Issues. Feel free to collaborate there as well!

**Reply by luisfrocha** [reply]
Currently using the iPad version on Vision Pro. Do you have a TestFlight for trying the AVP version?
